libshaderc/include/shaderc.h

```diff
@@ -128,6 +128,10 @@ void shaderc_compile_options_set_disassembly_mode(
 void shaderc_compile_options_set_preprocessing_only_mode(
     shaderc_compile_options_t options);
 
+// Sets the compiler to suppress all warning messages before they are output.
```

Comment:
In another CL we can fix the other "Sets the compiler to..." comments to be "Sets the compiler mode to..."